set.mm

```diff
@@ -20403,17 +20403,17 @@ Corresponds to the dual of Axiom (B) of modal logic.  (Contributed by NM,
 
   ${
     $d x y w $.  $d ph w $.  $d ps w $.  $d w z $.
-    sbievg.1 $e |- F/ y ph $.
-    sbievg.2 $e |- F/ x ps $.
-    sbievg.3 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    cbvsbvf.1 $e |- F/ y ph $.
+    cbvsbvf.2 $e |- F/ x ps $.
+    cbvsbvf.3 $e |- ( x = y -> ( ph <-> ps ) ) $.
     $( Substitution applied to expressions linked by implicit substitution.
        The proof was part of a former ~ cbvabw version.  (Contributed by GG and
        WL, 26-Oct-2024.) $)
-    sbievg $p |- ( [ z / x ] ph <-> [ z / y ] ps ) $=
+    cbvsbvf $p |- ( [ z / x ] ph <-> [ z / y ] ps ) $=
```

Comment:
Then the modification of the label should also be reflected in the comment: "Change bound variables in a wff substitution. ..." (cbv = "Change bound variables", sb = "substitution").